js_modules/dagster-ui/packages/ui-core/src/graph/layout.ts

```diff
@@ -134,7 +134,7 @@ export function layoutOpGraph(pipelineOps: ILayoutOp[], opts: LayoutOpGraphOptio
   }
 
   // Define a new top-down, left to right graph layout
-  g.setGraph({rankdir: 'TB', marginx, marginy});
+  g.setGraph({rankdir: 'TB', marginx, marginy, ranker: 'tight-tree'});
```

Comment:
we should probably make the op graph layout respect the feature flag too, I can do that in a follow-up though probably not necessary for now